### PR TITLE
Create zh_CN.lang

### PR DESCRIPTION
**Pull request by 3TUSK**
**Half way done, still a working-in-progress, don't merge this for now**

Essentially following the version provided by @crafteverywhere , he always gives us surprise on dictions, syntaxes and many other details.  
Also use all fullwidth punctuation.

To-do:
- [ ] Curiosity. I doubt it is related to the new research system, but I have no access to the beta version of TC6 so I cannot confirm.
- [ ] Primordial Mote, Primordial Nodule. Same as Curiosity.
- [ ] Caster's Gauntlet. Definitely related to the new Focus system.
- [ ] Structure Magic. Guess there is terminology floating around...
- [ ] Knowledge types. Same as Curiosity, citation needed, context needed...
- [ ] The *thaumonomicon*. 
- [ ] Cleanup [Westernized Chinese language occurrences](https://en.wikipedia.org/wiki/Westernised_Chinese_language)

Summon @J-swl @Tartaricacid @SihenZhang @Mrkwtkr for proofreading. I did some minor changes to several of translations.  
Constructive comments are welcome.

### CODE REVIEW

**Comment by Azanor** [comment]
Its cool that you guys are getting going with this so long, but just a word of warning - this is definitely subject to change. There is also a lot of old stuff in the lang file that will be removed at some point.

File: zh_CN.lang [hunk 1]
```diff
@@ -0,0 +1,1258 @@
...
+item.cluster.lead.name=铅原矿簇
+item.cluster.cinnabar.name=朱砂原矿簇
+item.cluster.gold.name=金原矿簇
+item.cluster.quartz.name=石英原矿簇
+item.filter.name=源质滤管
+item.morphic_resonator.name=形态谐振器
+item.tallow.name=魔力油脂
+item.void_seed.name=虚空种子
+item.salis_mundus.name=世界盐
+item.primal_charm.name=元始魔力
```
Comment (NatsuArashi) [inline zh_CN.lang]: 之前的版本這東西好像翻作符咒 雖然怪怪的

File: zh_CN.lang [hunk 2]
```diff
@@ -0,0 +1,1258 @@
...
+entity.thaumcraft.EldritchGuardian.name=邪术守卫
+entity.thaumcraft.EldritchWarden.name=邪术典狱官
+entity.thaumcraft.EldritchWarden.name.custom=%d典狱官%s
+entity.thaumcraft.EldritchGolem.name=邪术构装体
+entity.thaumcraft.EldritchGolem.name.custom=邪术构装体%s
+entity.thaumcraft.EldritchOrb.name=邪术球
+entity.thaumcraft.CultistKnight.name=血腥骑士
+entity.thaumcraft.CultistCleric.name=血腥教徒
+entity.thaumcraft.CultistLeader.name=血腥主教
+entity.thaumcraft.CultistLeader.name.custom=%d血腥主教%s
```
Comment (DLslime) [inline zh_CN.lang]: 这里TC的翻译一直有个问题，如果TC6在这里和前面的版本一样的话，这里的"%s"指的是490行~502行的“XXXX的”，是BOSS的定语；而“%d”是游戏给BOSS随机生成的名字。那么按照汉语的语序习惯就应该翻译为“%s典狱官%d”、“%s邪术构装体”和“%s血腥主教%d”

Comment (NatsuArashi) [inline zh_CN.lang]: https://crowdin.com/project/minecraft/discussions/3153 參考 可能還需要另外測試

Comment (DLslime) [inline zh_CN.lang]: @NatsuArashi 非常感谢您的提醒，根据我在1.8.9用TC5的测试，“%s邪术构装体”的改动完全没有任何问题，因为只有一个变量，但其它的两个就改了白改。使用%_$_后方才正常生效，所以应该改为“%2$s典狱官%1$d”和“%2$s血腥主教%1$d”

Comment (3TUSK) [inline zh_CN.lang]: [O, Formatter!](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html)

File: zh_CN.lang (hunk 1 above)
Comment (3TUSK) [inline zh_CN.lang]: 我也有这个记忆...

**Comment by 3TUSK** [comment]
@Azanor I believe it is easier to constantly make small update than to wait until last minute and make a huge update. That's also why I made this pull request.

File: zh_CN.lang [hunk 3]
```diff
@@ -0,0 +1,1258 @@
...
+
+
+tile.banner.cultist.name=血腥教徒旗帜
+tile.banner.white.name=白色旗帜
+tile.banner.orange.name=橙色旗帜
+tile.banner.magenta.name=品红色旗帜
+tile.banner.light_blue.name=淡蓝色旗帜
+tile.banner.yellow.name=黄色旗帜
+tile.banner.lime.name=黄绿色旗帜
+tile.banner.pink.name=粉色旗帜
```
Comment (DLslime) [inline zh_CN.lang]: 原版为“粉红色”

File: zh_CN.lang [hunk 4]
```diff
@@ -0,0 +1,1258 @@
...
+text.spa.mix.true=药浴
+text.spa.mix.false=仅以该液体洗浴
+#
+color.white=白色
+color.orange=橙色
+color.magenta=品红色
+color.light_blue=淡蓝色
+color.yellow=黄色
+color.lime=黄绿色
+color.pink=粉色
```
Comment (DLslime) [inline zh_CN.lang]: 原版为“粉红色”

File: zh_CN.lang [hunk 5]
```diff
@@ -0,0 +1,1258 @@
...
+tile.pedestal.normal.name=奥术基座
+tile.pedestal.eldritch.name=邪术基座
+tile.pedestal.ancient.name=荒古基座
+tile.nitor.white.name=白色闪耀之光
+tile.nitor.orange.name=橙色闪耀之光
+tile.nitor.magenta.name=品红色闪耀之光
+tile.nitor.light_blue.name=淡蓝色闪耀之光
+tile.nitor.yellow.name=黄色闪耀之光
+tile.nitor.lime.name=黄绿色闪耀之光
+tile.nitor.pink.name=粉色闪耀之光
```
Comment (DLslime) [inline zh_CN.lang]: 原版为“粉红色”

File: zh_CN.lang [hunk 6]
```diff
@@ -0,0 +1,1258 @@
...
+# Doodads
+tile.table_stone.name=石桌
+tile.table_wood.name=木桌
+tile.candle.white.name=白色油脂蜡烛
+tile.candle.orange.name=橙色油脂蜡烛
+tile.candle.magenta.name=品红色油脂蜡烛
+tile.candle.light_blue.name=淡蓝色油脂蜡烛
+tile.candle.yellow.name=黄色油脂蜡烛
+tile.candle.lime.name=黄绿色油脂蜡烛
+tile.candle.pink.name=粉色油脂蜡烛
```
Comment (DLslime) [inline zh_CN.lang]: 原版为“粉红色”

File: zh_CN.lang [hunk 7]
```diff
@@ -895,13 +897,13 @@ nodetype.4=污染
...
-book.start.1="I had the strangest dream last night.\n\nIn the dream I took three of those strange crystals I've been finding and ground them with a handful of redstone in a bowl using a piece of flint."
-book.start.2="The crystals had to be of different types, but if I did it properly then the result was a strange, glowing dust.\n\nIn the dream I took the dust and sprinkled it on a bookcase, but the dream ended before I saw what happened."
-book.start.3="I wonder, should I do what the dream showed?\n\nI have the impression the dust was going to reveal something wondrous, but dangerous."
+got.crystals=手捧水晶的你，指尖忽然感受到一阵刺痛。或许有什么特殊含义？或许别的什么将给你带来灵感。
+got.dream=你从异梦中醒来。你快速记下了转瞬即逝的记忆。
+false.start=差不多了，但感觉还差什么...
+book.start.title=异梦
+book.start.1=“昨夜的梦，甚是诡异。\n\n我清楚地记得我在梦中将三块水晶连同一份红石用燧石捣碎，用碗装起来。”
+book.start.2=“三块水晶的类型须得是互不相同。若如此做，我应当得到一种散发着微弱光芒的粉末。\n\n在梦中，我将粉末撒向了书架。但在我即将看到结果的时候，梦醒了。”
+book.start.3=“我在想，我要不要重演一遍梦境中的景象？\n\n直觉告诉我，这粉末将为我掀开一个惊艳而危险的世界的面纱。”
```
Comment (3TUSK) [inline zh_CN.lang]: For anyone wonder why I use “惊艳” (lit. surprisingly beautiful, according to my memory): _wondrous_ and _dangerous_ are in rhyme, so does “艳” (yàn | ㄧㄢˋ) and “险” (xiǎn | ㄒㄧㄢˇ) (approximately). 

> They have _almost the same_ [final](https://en.wikipedia.org/wiki/Middle_Chinese_finals)s in [Middle Chinese](https://en.wikipedia.org/wiki/Middle_Chinese)... well that doesn't really count, as their word meaning differ from what it is today.

Comment (limuness) [reply zh_CN.lang]: 惊艳而危险的世界的面纱，这2个的会不会读上去有点奇怪？

Comment (3TUSK) [reply zh_CN.lang]: @limuness 一开始是冲着押韵（仔细看上面的恢复回复），用了这两个词。有别的选择吗？

Comment (limuness) [reply zh_CN.lang]: 无其他疑问，这样就行了，再改这个也没大意义。

Comment (liach) [reply zh_CN.lang]: `的`字太多的确听起来别扭

话说这个坑还在填吗？我以为本模组早挂了

Comment (limuness) [reply zh_CN.lang]: @liach  恩 最近azanor又重新开始制作了在他的捐赠页面上于11月6号发布了a13版本，所以我特此来这看看本地化进度。

**Comment by 3TUSK** [comment]
“这粉末将指引我前往一个惊艳而危险的世界。”？

**Comment by limuness** [comment]
直觉告诉我，这粉末将引导我前往一个惊艳又危险的世界。 这样呢?! 3T你感觉如何

File: zh_CN.lang (hunk 7 above)
Comment (NatsuArashi) [reply zh_CN.lang]: 這邊引號不能改 會出事 那個引號是give book的command

Comment (3TUSK) [reply zh_CN.lang]: 居然 give book 的 command 的一部分吗... 感谢提醒，这就尽快更正

Comment (NatsuArashi) [reply zh_CN.lang]: 剛剛拿beta來測試時發現這本書怎麼改都會壞掉 
花了兩小時才找到問題
如果沒修好大家就不會玩了233

**Pull request merged**